README.md

```diff
@@ -21,7 +21,7 @@ If you're using ``Bundler``, just update your ``Gemfile`` as follows:
     gem 'ddtrace', :source => 'http://gems.datadoghq.com/trace/'
 ```
 
-### Quickstart (manual instrumentation)
+### Rails Quickstart (manual instrumentation)
```

Comment:
this is a Quickstart for general manual instrumentation. In particular the following example is related to a simple Sinatra application (simple == tutorial). I think it's not related to Rails, right?